codecov.yml

```diff
@@ -0,0 +1,2 @@
+codecov:
```

Comment:
`Codecov pull request comment is a highly detailed overview of a pull request`. is current config able to do that? As I don't think so

Reply:
I have generated the codecov token from https://codecov.io/gh/fossasia/query-server/settings
And only a Maintainer can access that token, as he will have push rights.

Reply:
@vaibhavsingh97 
![screenshot from 2018-01-08 18-00-22](https://user-images.githubusercontent.com/22936570/34670893-33dd2fde-f49e-11e7-9390-fe38863778eb.png)